src/app/api/api.py

```diff
@@ -93,7 +93,7 @@ def node_ids_for_major_collections(
         ...,
         examples={
             "Alle Fachportale": {"value": COLLECTION_ROOT_ID},
```

Comment:
The "value" is required by FastAPI/Starlette to show these values as examples inthe OpenAPI specs. I do not want to remove this, because it is good to see in the API.
I did, however, inline the create_examples

Reply:
Ah, ok! Nice :)